README.md

```diff
@@ -1,207 +1,249 @@
 - `instance segmentation`
 - `panoptic segmentation`
+- `multi-modal`
+- `similarity search`
+- `image captioning`
+- `generative`
+- `super resolution`
 
-This should align with the `label:tasks` values defined in [STAC Label Extension][stac-ext-label-props] for relevant
-STAC Collections and Items employed with the model described by this extension.
+If the task falls within the category of supervised machine learning and uses labels during training, this should align with the `label:tasks` values defined in [STAC Label Extension][stac-ext-label-props] for relevant
+STAC Collections and Items published with the model described by this extension.
 
 [stac-ext-label-props]: https://github.com/stac-extensions/label#item-properties
 
+#### Result Array Object
+
+| Field Name | Type      | Description                                                                                                                                                                                                                                                                                                                                                          |
+|------------|-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| shape      | [integer] | **REQUIRED.** Shape of the n-dimensional result array ($N \times H \times W$), possibly including a batch size dimension. The batch size dimension must either be greater than 0 or -1 to indicate an unspecified batch dimension size.                                                                                                                              |
+| dim_names  | [string]  | **REQUIRED.** The names of the above dimensions of the result array, ordered the same as this object's `shape` field.                                                                                                                                                                                                                                                |
+| data_type  | enum      | **REQUIRED.** The data type of values in the n-dimensional array. For model outputs, this should be the data type of the result of the model inference  without extra post processing. Use one of the [common metadata data types](https://github.com/radiantearth/stac-spec/blob/f9b3c59ba810541c9da70c5f8d39635f8cba7bcd/item-spec/common-metadata.md#data-types). |
+
+#### Class Object
+
+See the documentation for the [Class Object](https://github.com/stac-extensions/classification?tab=readme-ov-file#class-object). We don't use the Bit Field Object since inputs and outputs to machine learning models don't typically use bit fields.
```

Comment:
Since the "Class Object" (not "Classification") is referred here, the "Bit Field" mention is more misleading than anything. Should be removed since `classification:bitfields` are not contained in `classification:classes` anyway.

Reply:
https://github.com/rbavery/dlm-extension/pull/2